### PR TITLE
Data context

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added Data context to get Data from the Backend

1)Added Data context and Reducer to get data from the backend
2) Displayed that data on the home page
3)Refactored video Card Components 
4)Added Data in Db files and and also added images

### CODE REVIEW

File: src/frontend/services/dataServices.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+import { dataConstants } from "../constants";
+
+const getVideos = async (dispatch) => {
+	try {
+		const response = await axios.get("/api/videos");
+		dispatch({
```
Comment (Richa-rani02) [inline src/frontend/services/dataServices.js:7]: suggestion: u can extract all the URL in a separate file

**Review by Richa-rani02 — APPROVED** [review]
Suggestion: format your code before pushing .Everything else looks fine liked that you have defined all the constants separately

**Review by tanishk15sharma — APPROVED** [review]
Naming conventions is nice , using `_` differentiates your names . you can using a separate file for all relative `URL`

**Review by Mohitkumar33 — APPROVED** [review]

**Review by Manpreet-hub — APPROVED** [review]
code is clean &  well structured .

File: src/frontend/services/dataServices.js (hunk 1 above)
Comment (prathmeshjagtap) [reply src/frontend/services/dataServices.js:7]: will do that

**Pull request merged**